client/src/utils/media.ts

```diff
@@ -0,0 +1,26 @@
+import '@expo/match-media';
+
+import React from 'react';
+import { useMediaQuery } from 'react-responsive';
+
+type Props = {
+  children: React.ReactElement | null
+};
+
+export type MediaQueryType = {
+  desktop: boolean,
+  tablet: boolean,
+  mobile: boolean,
+}
+
+const isDesktop = (): boolean => useMediaQuery({ minWidth: 1224 });
+const isTablet = (): boolean => useMediaQuery({ minWidth: 768 });
+const isMobile = (): boolean => useMediaQuery({ minWidth: 0 });
+
+export const getMediaQuery = (): MediaQueryType => {
+  return {
+    desktop: isDesktop(),
+    tablet: isTablet(),
+    mobile: isMobile(),
+  };
+};
```

Comment:
```suggestion
export const useMedia = (): MediaQueryType => {
  const isDesktop = useMediaQuery({
    minWidth: 1224
  });
  const isTablet = useMediaQuery({
    minWidth: 768,
    maxWidth: 1223,
  });
  const isMobile = useMediaQuery({
    minWidth: 0,
    maxWidth: 767,
  });
  return {
    desktop: isDesktop,
    tablet: isTablet,
    mobile: isMobile,
  };
};
```

Reply:
@0916dhkim Nice updates! I just want to point out here that if we only want to consider `mobile` and the `tablet` or the bigger screens, could we avoid using `desktop` in our implementation?

```
const StyledScrollView = styled.ScrollView`
  align-self: center;

  ${({ theme: { mobile } }) => mobile && css`
    width: 100%;
  `}

  ${({ theme: { tablet } }) => tablet && css`
    width: 50%;
    max-width: 800;
  `}
`;
```

Reply:
I am afraid of using `minWidth` and `maxWidth` strictly, since the above isn't possible currently.